build.gradle.kts

```diff
@@ -29,7 +29,7 @@ buildscript {
 plugins {
     idea
     id("org.jetbrains.kotlin.jvm")
-    id("org.jetbrains.intellij.platform") version "2.6.1-SNAPSHOT"
+    id("org.jetbrains.intellij.platform") version "2.6.0"
```

Comment:
I hope that 2.7.0 will be released soon.
With 2.6.0 our `./gradlew testWithoutAgent` tests are broken. With 2.6.1-SNAPSHOT things became broken a few days ago. But https://github.com/getappmap/appmap-intellij-plugin/pull/900 should be fix the problems we currently have. #900 is preferred over rolling back to broken 2.6.0.